### ISSUE TITLE
Bug: Font Awesome Flag

### ISSUE THREAD

**Issue by SnarkyDeveloper** [FortAwesome/Font-Awesome#19618]
### Bug description

I was looking at the Font Awesome flag and saw this on duotone

### Reproducible test case

https://fontawesome.com/icons/font-awesome?s=duotone&f=classic

### Screenshots

![DE89E912-EBE2-4785-BF89-EFD34514FD78](https://user-images.githubusercontent.com/91441703/225704801-df30e1d6-b6ff-45f3-847c-5001605658a6.jpeg)

### Font Awesome version

v6.0.3

### Serving

Kit

### Implementation

Other (as specified in the bug description)

### Browser and Operating System

Mobile Chrome

### Web bug report checklist

- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

IIRC this is a known bug or it has been reported elsewhere, but at the moment I'm not able to find the original issue

Probably it is because font awesome flag belongs to both the standard and the brand sets

Let's assign @sensibleworld

**Comment by sensibleworld** [comment]
Yeah, this is a bug in our icon search related to icons that exist in both brands and our standard styles. The icons themselves will still work in all of our implementations.

**Comment by SnarkyDeveloper** [comment]
Ok thank you just trying to help

**Comment by sensibleworld** [comment]
This has been fixed.
